packages/frontend/src/components/common/Table/ScrollableTable/RichBodyCell.tsx

```diff
@@ -0,0 +1,34 @@
+import { ResultRow } from '@lightdash/common';
+import { Cell } from '@tanstack/react-table';
+import { FC } from 'react';
+import styled from 'styled-components';
+
+interface RichBodyCellProps {
+    cell: Cell<ResultRow>;
+}
+
+const Link = styled.a`
+    :hover {
+        text-decoration: none;
+    }
+`;
+
+export const isUrl = (value: string) => {
+    return (
+        value &&
+        typeof value === 'string' &&
+        (value.startsWith('http://') || value.startsWith('https://'))
```

Comment:
I don't want to overengineer, but also this is not a good solution.

wdyt should we treat for example `goo.gl` as an URL without protocol

Reply:
I think this approach is fine and force users to add a valid protocol.